MassEffectModManagerCore/modmanager/localizations/int.xaml

```diff
@@ -565,7 +565,7 @@
     <system:String x:Key="string_error">Error!</system:String>
     <system:String x:Key="string_fullGameRestore">Full game restore</system:String>
     <system:String x:Key="string_restoringTextureLODs">Restoring from backup will reset your texture quality settings back to the defaults. Changing the active boot target to one with ALOT installed will prompt you enable high resolution texture settings again.</system:String>
-    <system:String x:Key="string_entireGameDirectoryWillBeDeleted">The entirety of the game directory will be deleted. If you are trying to do granular restoration, including removing DLC mods, restoring specific files or SFARs, use the restore options from the Target Info button instead.</system:String>
+    <system:String x:Key="string_entireGameDirectoryWillBeDeleted">The entirety of the game directory will be deleted. If you are trying to do granular restoration, including removing DLC mods, restoring specific files, or SFARs, use the restore options from the Target Info button instead.</system:String>
```

Comment:
Hard to say on this as I use an either/or in this case for files OR sfars. (Specific SFARs or specific files). Maybe changing  order such as restoring specific files or sfars, or removing DLC mods... not sure. What do you think?

Reply:
Yes, you're right, I must have misunderstood at first.
Changing the order would work, or alternatively maybe `including removing DLC mods and restoring specific files/SFARs`